src/app/zap-templates/zcl/data-model/chip/scene.xml

```diff
@@ -32,8 +32,8 @@ limitations under the License.
     <item name="AttributeValueList" type="AttributeValuePair" array="true"/>
   </struct>
 
-  <domain name="CHIP"/>
-    <cluster>
+  <cluster apiMaturity="provisional">
+    <domain>CHIP</domain>
```

Comment:
This isn't right.

There's a separate domain for the configurator and for the cluster, no?   And the domain for the cluster is already "General" below the name on line 38.  So the domain bits here were fine.

Reply:
Figured this out just a few days ago - domain affects which top-level tab-thing the cluster lands under in the zap tool.

Reply:
Incidentally, we should probably clean this up as well because we seem to have a somewhat random split between general and chip, but that's quite far down the priority list imo

Reply:
^ above comments are purely informational, not requesting you do anything, so feel free to just resolve this.